src/main/resources/templates/signup.html

```diff
@@ -2,25 +2,30 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
+    <!-- Bootstrap CSS and JS -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
     <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.6.0/dist/umd/popper.min.js" integrity="sha384-KsvD1yqQ1/1+IA7gi3P0tyJcT3vR+NdBTt13hSJ2lnve8agRGXTTyNaBYmCR/Nwi" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.min.js" integrity="sha384-nsg8ua9HAw1y0W1btsyWgBklPnCUAFLuTMS2G72MMONqmOymq585AcH49TLBQObG" crossorigin="anonymous"></script>
-
+    <!-- Local CSS and JS -->
     <link href="/css/main.css" rel="stylesheet" type="text/css"/>
     <script src="/js/signUp.js" type="text/javascript"></script>
     <script src="/js/shared.js" type="text/javascript"></script>
 
-    <title>Home</title>
+    <title>TvTracker Sign-up</title>
 </head>
 <body>
-    <div class="form">
+<!-- Create account form -->
+    <div class="form createAccountForm">
+        <h5>Enter a username</h5>
         <input id="username" placeholder="Enter Username"><br><br>
+        <h5>Enter an email</h5>
         <input type="email" id="email" placeholder="Enter Email"><br><br>
+        <h5>Enter your birthdate</h5>
         <input type="date" id="birthdate" placeholder="Enter Birthdate"><br><br>
+        <h5>Enter a password</h5>
         <input type="password" id="password" placeholder="Enter Password"><br><br>
+        <h5>Confirm your password</h5>
```

Comment:
I'd leave it as it were.  Placeholder is a good way to combine label and entry .  It essentially puts the label inside of the entry field.  That makes for a simpler UI, and less clutter.